### DISCUSSION TITLE
Diagramas

### DISCUSSION THREAD

**Discussion by LuceroLuciano** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#112] [Ideas]
Hola explorers. 

Les comparto este Link: [https://proyecto-semestral.readthedocs.io/en/latest/6%20-%20Design.html](https://proyecto-semestral.readthedocs.io/en/latest/6%20-%20Design.html). Me parecieron  interesante los diagramas que se explican como: Vista lógica, vista de desarrollo, vista de proceso, vista física y escenarios.

**Comment by GloriaCoralCerecedo** [comment]
Hola, muchas gracias, oye sabes como debe ser el Modelamiento de Negocio, entiendo que es solo como el abogado maneja el sistema. Lo que mandaste para las demás vistas ya incluye todo con los usuarios y todo eso.  No se si estoy en lo correcto.
